modules/RTC/DataChannels.js

```diff
@@ -197,7 +197,13 @@ DataChannels.prototype._onXXXEndpointChanged = function (xxx, userResource) {
             jsonObject.colibriClass = (upper + 'EndpointChangedEvent');
             jsonObject[lower + "Endpoint"]
                 = (userResource ? userResource : null);
-            dataChannel.send(JSON.stringify(jsonObject));
+            //Tries to fix errors on peer connection close.
+            try {
+                dataChannel.send(JSON.stringify(jsonObject));
+            } catch (e) {
+                // FIXME: Handle properly
```

Comment:
If the peer connection is closed the user shouldn't use the data channels. But even if they are used that way they won't brake anything by stopping the execution. 

I was just thinking that we can maybe find more gentle solution in future. But this seems to me not directly related to the current PR.